### DISCUSSION TITLE
(When) we can expect real fuel consumption? :)

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#2851] [Q&A] [answered]
First of all - I am so thankful to the whole FlyByWire-Team for making the Sim enjoyable and the "stock" A320 flyable - and everyday a little bit more. Awesome project and awesome community!
I try to keep it short and simple:
Me as a real life Airbus pilot, and to my mind all other here wants to make a flight "as real as it gets". And one of the basic and fundamental things in terms a commecing a flight is the fuel planning. 
Sadly, it is not really possible to do it yet with the stock A320 and even the FBW version because the fuel consumption is not close to the reality. I think it is basically a core FS problem. I read that FBW develop a new engine model that is more realistic.
And I know that it takes time... and a lot of time to make it good! I really know that everything here is hard work to make it better then ASOBO does it.
I only want to know, if somebody knows the state of this development - when can we expect realistic fuel consumption within the A32NX? Please, thats only a question that which may not be answered either. Thats not a post to make FBW work faster or something like this. Thats only because i really looking forward to that! :) 

Again - THANK YOU, FlyByWire!

**Comment by Lucky38i** [comment, marked as answer]
Not the answer you want but.. When it's ready. No one here works on a deadline, we only work when we've got freetime so things are released when we feel like it's at a ready state.

**Reply by andrejetcat** [reply]
the best thing to know, that IT IS BEING WORKED ON! :) 
Please let me know, when you need something - We have some NEO´s in our fleet, so I may get some deeper information you need for your work.
